lib/exercises.js

```diff
@@ -1,17 +1,24 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n log n), where n is the combined length of the input strings (since the strings are being sorted)
```

Comment:
But you're not sorting an array of all the characters in all the strings.  

Instead I would say that the time complexity is O(n * m log m), where n is the number of strings and `m` is the length of the largest string.  If you could assume the all the strings were less than a certain number of characters you could then say it's O(n).